flang/docs/Intrinsics.md

```diff
@@ -657,6 +657,11 @@ CALL CO_REDUCE
 CALL CO_SUM
 ```
 
+### Library subroutine 
```

Comment:
Please put this new section later, after all of the non-standard intrinsics, so the actual intrinsic procedures appear together.